types/jest/index.d.ts

```diff
@@ -175,67 +177,71 @@ declare namespace jest {
         stringMatching(str: string | RegExp): any;
     }
 
-    interface Matchers {
+    interface Matchers<R> {
```

Comment:
why is this generic? i only see one use of it, and it takes `void`.
can you share references to the docs that clarify this change?